MvvmCross/Navigation/MvxNavigationService.cs

```diff
@@ -28,14 +28,15 @@ public class MvxNavigationService : IMvxNavigationService
         private IMvxViewDispatcher _viewDispatcher;
         public IMvxViewDispatcher ViewDispatcher
         {
-            get => _viewDispatcher ?? (IMvxViewDispatcher)MvxMainThreadDispatcher.Instance;
+            get => _viewDispatcher ?? (_viewDispatcher = Mvx.Resolve<IMvxViewDispatcher>());
```

Comment:
This didn't use to work for me. Are you sure that returns the correct instance from startup?